js/utilities.js

```diff
@@ -140,6 +140,154 @@ var ERMrest = (function(module) {
     };
 
     /**
+     * @function
+     * @param {Object} value A boolean value to transform
+     * @param {Object} [options] Configuration options
+     * @return {string} A string representation of a boolean value
+     * @desc Formats a given boolean value into a string for display
+     */
+    module._printBoolean = function(value, options) {
+        options = (typeof options === 'undefined') ? {} : options;
```

Comment:
This `options` line will likely be refactored out to the larger function (`Column.formatvalue`) that will call these print utility functions when that larger function is implemented.